pgcli/main.py

```diff
@@ -365,7 +365,7 @@ def connect(self, database='', host='', user='', port='', passwd='',
         # If we successfully parsed a password from a URI, there's no need to
         # prompt for it, even with the -W flag
         if self.force_passwd_prompt and not passwd:
-            passwd = click.prompt('Password', hide_input=True,
+            passwd = click.prompt('Password [%s]' % user, hide_input=True,
```

Comment:
One tiny nitpick can you add the word `for` before the user name in the square brackets? 
eg: 
```
    passwd = click.prompt('Password [for %s]' % user, hide_input=True,
```
This is to avoid the user from thinking that we're showing some default value.

Reply:
If you're adding a `for`, I think it's better to remove the `[]`. `Password for XXX` will look better, more like a coherent phrase.

Reply:
Sure thing, in a minute.

Reply:
Done.